From bc7616fb719f58690d0bbe351a7ce101c74dc7de Mon Sep 17 00:00:00 2001
From: Seongjun Hong <hsj0512@snu.ac.kr>
Date: Wed, 1 Jul 2026 12:55:01 +0000
Subject: [PATCH 04939/15333] docs: ABI: README: fix grammatical error

The Note describing how to write ABI documentation fields contained
a typo. "The fields should be use a simple notation" should be
"The fields should use a simple notation".

Signed-off-by: Seongjun Hong <hsj0512@snu.ac.kr>
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Message-ID: <20260701125501.93170-1-hsj0512@snu.ac.kr>
---
 Documentation/ABI/README | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/ABI/README b/Documentation/ABI/README
index 315fffe1f831..27c962e6a872 100644
--- a/Documentation/ABI/README
+++ b/Documentation/ABI/README
@@ -62,7 +62,7 @@ Users:		All users of this interface who wish to be notified when
 
 
 Note:
-   The fields should be use a simple notation, compatible with ReST markup.
+   The fields should use a simple notation, compatible with ReST markup.
    Also, the file **should not** have a top-level index, like::
 
 	===
-- 
2.55.0

